devel/mold/Portfile

```
@@ -0,0 +1,20 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+version             1.5.1
+
+github.setup        rui314 mold v1.5.1
```

The format for `github.setup` is:
```
github.setup  <author> <project> <version> <version_prefix> <version_suffix>
```

So the correct `github.setup` line would look like:
```suggestion
github.setup        rui314 mold 1.5.1 v
```

With this, you can delete that redundant `version` line above (line 6). You should also then be able to delete the `distname` line as well, as both of those are being set automatically by `github.setup`.  `homepage` too.